src/main/scala/org/constellation/storage/NotificationService.scala

```diff
@@ -9,16 +9,10 @@ import org.constellation.storage.algebra.{Lookup, MerkleStorageAlgebra}
 class NotificationService[F[_]: Concurrent]() extends MerkleStorageAlgebra[F, String, PeerNotification] {
 
   val merklePool =
-    new ConcurrentStorageService[F, Seq[String]](
-      ConstellationExecutionContext.createSemaphore(),
-      "notification_merkle_pool".some
-    )
+    new ConcurrentStorageService[F, Seq[String]](ConstellationExecutionContext.createSemaphore(), "notification_merkle_pool".some)
 
   val memPool =
-    new ConcurrentStorageService[F, PeerNotification](
-      ConstellationExecutionContext.createSemaphore(),
-      "notification_mem_pool".some
-    )
+    new ConcurrentStorageService[F, PeerNotification](ConstellationExecutionContext.createSemaphore(), "notification_mem_pool".some)
```

Comment:
I guess these won't be readjusted by scalafmt when someone runs it. I mean these aren't too long? :)

Reply:
You're right, they will. Intellij apparently ignores scalafmt when doing refactoring. Fixing it.